packages/api/endpoints/executions.js

```diff
@@ -257,7 +234,7 @@ async function del(req, res) {
 async function searchByGranules(req, res) {
   const payload = req.body;
   const knex = await getKnexClient();
-  const granules = await getGranulesForPayload(payload, knex);
```

Comment:
I can't find the getGranulesForPayload change,  is this just an update that got missed in a previous change?